lib/hobo_fields/model.rb

```diff
@@ -6,7 +6,7 @@ module HoboFields
   module Model
     class << self
       def mix_in(base)
-        base.singleton_class.prepend ClassMethods
+        base.singleton_class.prepend ClassMethods unless base.singleton_class < ClassMethods # don't mix in if a base class already did it
```

Comment:
This was a bug I'd introduced in my refactor. I always thought `prepend` and `include` were idempotent, so I'd removed the `unless`. Yes, they are, but only immediately. If any the `ancestors` chain had changed in the mean time (`inheritance` or another `prepend` or `include`), the ClassMethods would get mixed in again. For example, consider our base class `Organization`. It could have `fields do` which would mix in `ClassMethods`, then a derived class like `Network` would wind up mixing in `ClassMethods` a second time. So, for example, when `ClassMethods#belongs_to` called `super`, it wound up calling back into itself. And it was _not_ idempotent...because mutation. TIL.